commands/config.go

```diff
@@ -124,7 +125,12 @@ func (cmd *configCommand) args() (res []string, switchedToTLS bool) {
 		res = append(res, fmt.Sprintf("--paths-base=%s", cmd.BasePath))
 	}
 
+	if cmd.AgentPassword != "" {
+		res = append(res, fmt.Sprintf("--agent-password=%s", cmd.AgentPassword))
```

Comment:
Is this safe if the password has special chars like quotes or spaces?
Do we have to consider using `%q` instead of `%s`?

Reply:
@percona-csalguero it will work as expected becase it will add to `res` (array) another item, later it will forkProcess and pass that array, so we don't need to escape